avocado/utils/sysinfo.py

```
@@ -168,9 +169,15 @@ def collect(self):
         # but the avocado.utils.process APIs define no timeouts as "None"
         if int(self.timeout) <= 0:
             self.timeout = None
+
+        cmd_run =  self.cmd
```

Flake8 (7.2.0)

[error] multiple spaces after operator

(E222)